### DISCUSSION TITLE
Hololens 1 Development Stack

### DISCUSSION THREAD

**Discussion by loibnerg** [microsoft/MixedRealityToolkit-Unity#10995] [Q&A] [answered]
I'm a teacher at a higher technical school in Austria and we currently have 3 Hololens 1 devices. Currently we work with these devices with following stack:

Unity 2019.4.40 with legacy XR
MRTK 2.8.3
Visual Studio 2022

Is this still the recommended configuration to work with Hololens 1 or should we change something?

I'm looking forward to your answers.

Best wishes, Gernot

**Comment by keveleigh** [comment, marked as answer]
Hello! That's probably still the best recommended stack for HoloLens 1 devices. You _could_ move to Windows XR Plugin if you wanted to, especially if you have plans to use other devices and/or newer Unity versions in the future, but if you're staying on Unity 2019, legacy XR should work for you.
